primer/brand
# Remove redundant styles in Section and BreakoutBanner components

## Summary

Fixes issues found in [recent release testing](https://github.com/primer/brand/pull/661#issuecomment-2265106149) for Section and BreakoutBanner components



## List of notable changes:



- removes redundant / incorrect inline styles from the default Section and Breakout Banner components
- fixes a typo in Storybook story name for Section


## Steps to test:



1. Use inspector to verify the 'Default' story doesn't show inline styles for `undefined` values



## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2024-08-02 at 16 02 04](https://github.com/user-attachments/assets/f31b2909-2da5-4413-89b6-7cacba8666b2)


 </td>
<td valign="top">

![Screenshot 2024-08-02 at 16 06 01](https://github.com/user-attachments/assets/b6919fcb-8c86-4ffc-8e1e-b3c166403fe9)

</td>
</tr>
</table>
